content/specs/atp.md

```diff
@@ -1,294 +1,82 @@
-
-If you expect users have used their primary password with your client, we also strongly encourage you to delete all existing refresh tokens and re-fetch access/refresh tokens using an app password.
-
-App passwords are of the form `xxxx-xxxx-xxxx-xxxx`. For your users' safety, you could run a quick check to ensure that they are logging in with an app password and not their account password.
-
-For users to generate an app password, navigate to Settings > Advanced > App passwords.
-
-### Atproto core lexicon
-
-The com.atproto.* lexicons provides the following behaviors:
+## Protocol Structure
 
-- [com.atproto.identity](/lexicons/com-atproto-identity). Handle resolution and changes.
-- [com.atproto.server](/lexicons/com-atproto-server). Account and session management.
-- [com.atproto.moderation](/lexicons/com-atproto-moderation). Moderation reporting.
-- [com.atproto.repo](/lexicons/com-atproto-repo). Repo CRUD operations.
-- [com.atproto.sync](/lexicons/com-atproto-sync). Repo content sync and streaming.
+**Identity:** account control is rooted in stable [DID](/specs/did) identifiers, which can rapidly resolved to determine the current service provider location and cryptographic keys associated with the account. [Handles](/specs/handle) provide a more human-recognizable and mutable identifier for accounts.
 
-### Additional lexicons
+**Data:** public content is stored in content-addressed and cryptographically verifiable [Repositories](/specs/repository). data records and network messages all conform to a unified [Data Model](/specs/data-model) ([IPLD](https://ipld.io/docs/data-model/), with [CBOR](https://en.wikipedia.org/wiki/CBOR) and JSON representations).
 
-For atproto to be practically useful, it needs to support a variety of sophisticated queries and behaviors. While these sophisticated behaviors could be implemented on the user device, doing so would perform more slowly than on the server. Therefore, the PDS is expected to implement lexicons which provide higher-level APIs. The reference PDS created by Bluesky implements the [bsky.app lexicon](/lexicons/bsky-app).
+**Network:** HTTP client-server and server-server [APIs](/specs/xrpc) are described with Lexicons, as are WebSocket [Event Streams](/specs/event-stream). Individual records can be referenced across the network by [AT URI](/specs/at-uri-scheme). A Personal Data Server (PDS) acts as an account's trusted agent in the network, routes client network requests, and hosts repositories. A Big Graph Server (BGS) crawls many repositories and outputs a unified event firehose.
 
-## Server-to-server API
+**Application:** APIs and record schemas for applications built on atproto are specified in [Lexicons](/specs/lexicon), which are are referenced by [Namespaced Identifiers](/specs/nsid) (NSIDs). Application-specific aggregations (such as search) are provided by an Application View (AppView) service. Clients can include mobile apps, desktop software, or web interfaces.
```

Comment:
Hmm! AppView is new to me. Sounds like it overlaps a lot with BGS. Is BGS the architectural concept, and AppView is more of a service/interface?

Reply:
The BGS is primarily a "dumb" rehoster of content across PDSes, whereas the AppView provides the views specific to the Bluesky application.  The BGS doesn't need to have semantic awareness of the content it's hosting, whereas the AppView is interpreting "like", "follow", "post" records into more complex views to service an application.  Roughly speaking, BGS is concerned with `com.atproto.sync.*` and `com.atproto.repo.*` lexicons, and the Bluesky AppView is concerned with `app.bsky.*` lexicons.